bot/extensions/youtube/tasks.py

```diff
@@ -44,6 +44,11 @@ async def check_for_new_videos(self):
         await self.check_old_thumbnails()
         await self.find_new_videos()
 
+    @check_for_new_videos.error
+    async def on_check_error(self, error: Exception):
+        """Logs any errors that occur during the check_for_new_videos task"""
+        await self.bot.on_error("check_for_new_videos", error)
```

Comment:
Instead of using `bot.on_error` we should call `bot.send_error` directly so we don't have to handle overwriting the base function and adding the logic to handle the different function signatures.